collect-connect.rst

```diff
@@ -0,0 +1,33 @@
+Connecting to a Server
+================================
+
+ODK Collect is used to complete surveys with participants. But if you want to review and analyse your collected data, you'll need to upload your collected data to another application.
+
+.. note::
+
+  When you first install Collect, it connects to the `ODK Aggregate Demo server <https://opendatakit.appspot.com/Aggregate.html>`_. This allows you to try out the app by :ref:`downloading blank example forms <in-app-get-blank-forms>`, :doc:`filling them out   <collect-filling-forms>`, and :ref:`uploading completed forms <uploading-forms>` back to the demo server.
```

Comment:
Inconsistent usage of Collect vs ODK Collect. It'd be good to do a pass through the text for consistency.

Reply:
We have a note about this in the contrib guide, but it leaves a lot of wiggle room.